src/components/GroupedTaskStatusBadge/index.tsx

```diff
@@ -78,6 +75,7 @@ interface BadgeColorProps {
   border?: string;
   fill?: string;
   text?: string;
+  isActive?: boolean;
 }
 
 const BadgeContainer = styled.div<BadgeColorProps>`
```

Comment:
When I click "Lint" build variant, the task badges become inactive, but I would expect them to be active since they're part of the build variant I'm looking at:

<img width="304" alt="Screenshot 2023-06-15 at 1 04 41 PM" src="https://github.com/evergreen-ci/spruce/assets/47064971/62e57307-461f-4c97-8c1e-e899171c8cd4">

Follow-up question: should these text links have different visual appearances in inactive versus active states?

Reply:
Oh wow totally missed this great catch!